shell/platform/embedder/embedder.h

```diff
@@ -2434,6 +2434,41 @@ typedef struct {
   /// being registered on the framework side. The callback is invoked from
   /// a task posted to the platform thread.
   FlutterChannelUpdateCallback channel_update_callback;
+
+  /// The version of the app (e.g., 1.0.0). Used by Shorebird.
+  ///
+  /// The string can be collected after the call to `FlutterEngineInitialize`
+  /// returns. The string must be NULL terminated.
+  const char* app_version;
```

Comment:
For maintainability during further merges, perhaps prepare a separate struct and have a pointer to it here?

Reply:
That's a great idea. Will do

Reply:
Done